agent/agent-tooling/src/main/java/io/opentelemetry/sdk/metrics/internal/view/PreAggregatedStandardMetricsAttributesProcessor.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.opentelemetry.sdk.metrics.internal.view;
+
+import static com.azure.monitor.opentelemetry.exporter.implementation.AiSemanticAttributes.IS_PRE_AGGREGATED;
+
+import com.azure.monitor.opentelemetry.exporter.implementation.AiSemanticAttributes;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.sdk.trace.ReadableSpan;
+import java.util.Set;
+import java.util.function.BiConsumer;
+
+@SuppressWarnings("rawtypes")
+class PreAggregatedStandardMetricsAttributesProcessor extends AttributesProcessor {
+
+  private final Set<AttributeKey> view;
+  private final boolean includeSynthetic;
```

Comment:
Could you explain the `synthetic` notion? Perhaps a comment in the code?

Reply:
will add comment here:
>   public static final AttributeKey<Boolean> IS_SYNTHETIC =
      booleanKey("applicationinsights.internal.is_synthetic");